### ISSUE TITLE
Windows support 

### ISSUE THREAD

**Issue by codeskyblue** [iOSForensics/pymobiledevice#7]

**Comment by ohadcn** [comment]
The tools and the library should be fully supported on windows.
Have you encountered any specific problem with it?

**Comment by codeskyblue** [comment]
just can not run

**Comment by ohadcn** [comment]
What errors do you get? did you install the requirements from requirements.txt file? what command do you use to run it? (you can give my fork a try, it's an older version but easier to use)

**Comment by codeskyblue** [comment]
```
(master)*$ python afc.py                                                                   x
Traceback (most recent call last):                                                         x
  File "afc.py", line 34, in <module>                                                      x
    from construct.macros import String, ULInt64                                           x
ImportError: No module named macros
```

**Comment by GotoHack** [comment]
Fixed with commit 55d26b9f.
